docs/Manual/04 Fusing.md

```diff
@@ -12,18 +12,17 @@ import accera as acc
 schedule = acc.fuse(schedule0, schedule1, ...)
 ```
 
-*Full fusing* is the most straightforward form of fusing, where each dimension is fused with the corresponding dimension from the other schedules.
+*Full fusing* is the most straightforward fusing where each dimension is fused with the corresponding dimension from other schedules. 
```

Comment:
"... is the most straightforward, where each dimension is fused with the corresponding dimension from the other schedules."